benchmarks/opacities_opacity.py

```diff
@@ -14,95 +12,25 @@ class BenchmarkMontecarloMontecarloNumbaOpacities(BenchmarkBase):
     Class to benchmark the numba opacities function.
     """
 
-    @parameterize(
-        {
-            "Electron number density": [
-                1.0e11,
-                1e15,
-                1e5,
-            ],
-            "Energy": [
-                511.0,
-                255.5,
-                511.0e7,
-            ],
-        }
-    )
-    def time_compton_opacity_calculation(self, electron_number_density, energy):
-        compton_opacity_calculation(energy, electron_number_density)
+    def time_compton_opacity_calculation(self):
+        energy = 511.0
```

Comment:
511keV is quite low for the Compton opacity (almost at the Thomson limit), though it might not matter in this case.